manifest/examples/prometheus/rules.yml

```diff
@@ -1,8 +1,8 @@
 groups:
 - name: K8up
   rules:
-  - alert: baas_last_errors
-    expr: baas_backup_restic_last_errors > 0
+  - alert: k8up_last_errors
```

Comment:
This may break the example configuration, as the prometheus metrics are still using baas.

Reply:
Does this relate to the entire change set or only to these two lines?

What would you suggest? Should we close the entire PR, or only undo this specific change?

Reply:
It's only those two lines.

Reply:
I undid the changes on those two lines.